### ISSUE TITLE
Laravel 5.5 method pluck no working whit oracle

### ISSUE THREAD

**Issue by jescobedoD** [yajra/laravel-oci8#383]
I have implemented the following method.
  
when it is connected to mysql this method returns data, but when it is configured oracle does not return data
 
    public static function getRegionList(){

        $regiones  = Region::pluck('REG_DESCRIPCION','REG_ID'); 
        $regiones->prepend('Todas' , 0);

        return $regiones;
    }
 

### System details

 
![image](https://user-images.githubusercontent.com/6155599/33666876-0dae9e1c-da7a-11e7-8036-d761ce0596e4.png)

- Operating System window 10
- PHP Version  7.1.10
- Laravel Version  5.5
- Laravel-OCI8 Version 5.5.*

**Comment by yajra** [comment]
Thanks for reporting, will check this as soon as I can.

**Comment by yajra** [comment]
I just tried this and works well for me:

```php
$users = App\User::pluck('email', 'id');
dd($users->prepend('Todas', 0));
```

Output

```
Collection {#884 ▼
  #items: array:2 [▼
    0 => "Todas"
    1 => "admin@example.com"
  ]
}
````

**Comment by mstaack** [comment]
works for me too. must be a problem on your side.

**Comment by jescobedoD** [comment]
I found that the error was generated after the following case:

I have deginida the table with the names of the uppercase fields REG_ID when making the query pluck ('REG_ID'), I got 0 results. but pluck ('reg_id') worked without problems.

the system was first developed with mysql in which the error was not presented with the uppercase fields was only when the database was changed to oracle.

          public static function getRegionList(){

            $regiones  = Region::pluck('reg_descripcion','reg_id');
            $regiones->prepend('Todas' , 0);
            return $regiones;
          }
     
Thank you
